lib/routes/zjut/news.ts

```diff
@@ -0,0 +1,89 @@
+import { Route } from '@/types';
+import { load } from 'cheerio';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { parseDate } from '@/utils/parse-date';
+
+const host = 'http://www.news.zjut.edu.cn';
+
+const map = new Map([
+    [1, { id: '5414/list.htm', title: '图片新闻 - 浙江工业大学' }],
+    [2, { id: '5415/list.htm', title: '工大要闻 - 浙江工业大学' }],
+    [3, { id: '5416/list.htm', title: '综合新闻 - 浙江工业大学' }],
+    [4, { id: '5422/list.htm', title: '学术·探索 - 浙江工业大学' }],
+    [5, { id: '5423/list.htm', title: '三创·人物 - 浙江工业大学' }],
+    [6, { id: '5424/list.htm', title: '智库工大 - 浙江工业大学' }],
+    [7, { id: '5425/list.htm', title: '美誉工大 - 浙江工业大学' }],
+    [8, { id: '5419/list.htm', title: '葵园融媒 - 浙江工业大学' }],
```

Comment:
I would recommend allowing the user to enter real IDs like `5414` or `5415`, instead of mapped IDs.